src/site/content/en/blog/badging-api/index.md

```diff
@@ -210,8 +210,9 @@ Planning to use the App Badging API on your site? Your public support helps the
 Chrome team to prioritize features, and shows other browser vendors how critical
 it is to support them.
 
-* Send a Tweet to [@ChromiumDev](https://twitter.com/chromiumdev) tagged with
-  `#badgingapi` and let us know where and how you're using it.
+* Send a tweet to [@ChromiumDev](https://twitter.com/chromiumdev) tagged with the hashtag
```

Comment:
```suggestion
* Send a tweet to [@ChromiumDev](https://twitter.com/chromiumdev) using the hashtag
```

Reply:
Made this call to action consistent in https://github.com/GoogleChrome/web.dev/pull/4722/commits/a2da5b8fc1d23310762c809a007b709e24c1a909.